packages/ui/src/components/pressable/pressable.native.tsx

```diff
@@ -95,9 +80,35 @@ const PressableBase = forwardRef<PressableElement, PressableBaseProps>(
       onLongPress?.(event);
     }
 
-    return <RNPressable ref={ref} onPress={handlePress} onLongPress={handleLongPress} {...props} />;
+    return (
+      <PressableCore
+        ref={ref}
+        onPress={handlePress}
+        onLongPress={handleLongPress}
+        onPressIn={onPressIn}
+        onPressOut={onPressOut}
+        style={[pressEffectStyle, style]}
+        {...rest}
+      />
+    );
   }
 );
 
-export const Pressable = Animated.createAnimatedComponent(PressableBase);
-export type PressableProps = AnimatedProps<PressableBaseProps>;
+Pressable.displayName = 'Pressable';
+
+/**
+ * Press effect preset to mimic react-native Touchable transition.
+ * This preset is up for removal. It's only used for components that previously used Touchable, and require a new design for the pressed state.
```

Comment:
Is there a task open for this new design work we can link here?

Reply:
Good call. None at the moment, we have few scattered conversations here and there, I'll make one and follow up.

Reply:
If there isn't an updated one yet I'd stop calling it `legacy`

Reply:
Thanks @pete-watters, wrote a [linear issue](https://linear.app/leather-io/issue/LEA-1859/press-effects-for-legacy-touchableopacity-elements) and linked in the comment. Re: `legacy`, would marking it as `@deprecated` be more appropriate you think?

Reply:
great idea!